### ISSUE TITLE
Team Review Spooketti grading Dema

### ISSUE THREAD

**Issue by ShakeSphereStuff** [spooketti/BinaryProj#2]
The team should have a Web Page(s) that teach/explain: Sorting, Timing, Compares Swaps, etc.
Team Review ticket containing key Team and individual contributions

Growth/Accomplishments in work is according to historical Team Plan, or they show revisions to plan according to work:  Dema’s plan for executing from text to binary showed some knowledge of javascript, with a more heavy evidence 

Short falls/Improvement that could be made in Work or Team Plan, team highlights next steps or improvements that could be made: Not much shown, as he admitted, had free time during CSP, shows lack of risk

Showing key accomplishments according to requirement in Java Backend such as API, Abstract Class(es), Inheritance, Polymorphic behavior, Sorting Algorithms, etc. In Sorting, there should NOT be static classes or static methods.: Dema showed that he could display both a javascript frontend and a much cleaner CSS backend

Showing key user interaction and learning(s). For instance how you visualized Sorting Algorithm: It shows to the user how numbers and 

Per check.: 0.82/1

Freeform comment:
While this is refined and polished, I would have liked to see you expand on the idea, have decoders from binary to text, binary to decimal, have them integrated. While you have a beautiful frontend, I would have liked to see more javascript to be executed instead of parseInt() and .toString() for the translation. 
TLDR; Should have more javascript based content as a way to obtain a higher grade
